R/etwfe.R

```diff
@@ -219,8 +226,19 @@ etwfe = function(
   
   ## Estimation ----
   
-  ## Full formula
-  Fml = Formula::as.Formula(paste(lhs, " ~ ", rhs, "|", fes)) 
+  ##### NEW: demean the interacted variable ######
+  if (!is.null(intvar)){
+    intvar_dm <- data[intvar] - mean(dplyr::pull(dplyr::distinct(dplyr::filter(data, .Dtreat == T), get(id), .keep_all = T), intvar), na.rm = T)
```

Comment:
Another undeclared (in the DESCRIPTION file) tidyverse dependency . Realistically, I'm not going to bring in the whole of dplyr (and all its dependencies) for a single line of code that could be done pretty easily with base R (or fixest::demean).  Stepping back, can you say a bit about why this is using a different demeaning approach than the group-level demeaning approach we use for the regular covariates? Apologies if I have misunderstood something.